### ISSUE TITLE
S03.01-Exercise-RecyclerView - TODO (46)

### ISSUE THREAD

**Issue by florin-nica** [udacity/ud851-Sunshine#116]
// TODO (46) Instead of setting the text to "", set the adapter to null before refreshing

I think TODO text should be changed to: 
"set weather data to null using mForecastAdapter.setWeatherData" or "set adapter items to null", 
because setting adapter to null causes NullPointerException

**Comment by den-kozlov** [comment]
I Agree. TODO text is a bit off-putting. I couldn't understand why no data is displayed after Refresh button pressed when I did
mRecyclerView.setAdapter(null);
instead of
mForecastAdapter.setWeatherData(null);

That's how I understood the phrase "set the adapter to null".
